easybuild/easyconfigs/g/Godon/Godon-20210913-x86_64.eb

```diff
@@ -0,0 +1,42 @@
+##
+# This is a contribution from SIB Swiss Institute of Bioinformatics
+# Homepage:     https://www.sib.swiss/research-infrastructure/competence-centers/vital-it
+#
+# Authors::     Sebastien Moretti <sebastien.moretti@sib.swiss>
+#
+# Compilation with GO looks to use a broken dependency
+#
+# SO use the pre-compiled binaries!
+##
+
+easyblock = 'Binary'
+
+name = 'Godon'
+version = '20210913'
+versionsuffix = '-x86_64'
+
+software_license = 'LicenseGPLv3'
+
+homepage = 'https://bitbucket.org/Davydov/godon/'
+description = """Godon is codon models software written in Go."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://bitbucket.org/Davydov/godon/downloads']
+sources = ['godon-master-linux-gnu-x86_64']
```

Comment:
No versioned download? :-/

At the very least, let EasyBuild rename the file on download, something like:

```python
sources = [{
    'download_filename': 'godon-master-linux-gnu-x86_64',
    'filename': 'godon-%(version)s-linux-gnu-x86_64',
}]
```

It's much better to use a download URL or filename that includes a version though, if that's available, since otherwise the download will be broken due to getting a different checksum if there's an update...

Reply:
the source code is available too (it's written in `Go`), so we could look at a from-source build too. Looks like it relies on `NLOpt` and a `BLAS` implementation

Reply:
I tried to compile it several times, but it always failed in my hands, as most Go packages do.

Reply:
There is no version for the binary download, that's why I used the last change date

Reply:
Building from source would indeed be better, but the versionsuffix makes it very clear we're using a pre-built binary here, so no blocker to merge this